tabnav header micro frontend (#3253)
SAP/luigi: docs/navigation-parameters-reference.md

### tabNav
- **type**: boolean or Object

> - **description**: renders the children of the node as a horizontal navigation bar. Sub-children are not supported. When you categorize nodes you will get a drop-down menu in the horizontal navigation. Set to `true` to show the horizontal navigation, or use the extra attributes to for more customization. 

Reviewer: ```suggestion
- **description**: renders the children of the node as a horizontal navigation bar. Sub-children are not supported. When you categorize nodes, you will get a drop-down menu in the horizontal navigation. Set to `true` to show the horizontal navigation, or use the extra attributes for more customization. (**since**: v0.7.0)
```

- **attributes**:

>   - **hideTabNavAutomatically**: boolean. In the case the node has only one child, it's possible to configure if the horizontal navigation bar will be hidden automatically or not. Set to true to hide the horizontal navigation bar and false otherwise. ( **since**: v0.7.0 )

Reviewer 2: I think hideTabNavAutomatically was added in v0.7.0. Can you re-edit the suggestion accordingly if thats the case

>   - **showAsTabHeader**: boolean. if set on the node, it considers it as horizontal navigation header microfrontend. The node should be webcomponent based and it should have nested children to show on the horizontal navigation bar. (**since**: NEXTRELEASE )

Reviewer: ```suggestion
  - **showAsTabHeader**: boolean. If this attribute is set on the node, it will be considered as a horizontal navigation header micro frontend. The node should be [webcomponent-based](web-component.md) and it should have nested children to show on the horizontal navigation bar. (**since**: NEXTRELEASE )
```

 children: [
  ...
                
// With hiding tab nav automatically if node has only one child               
  pathSegment: 'example',
  label: 'Example',
  tabNav: { hideTabNavAutomatically: true },
  children: [
  ...

  // showing horizontal navigation header microfrontend       
  pathSegment: 'header',
  label: 'Header Microfrontend',
  viewUrl: '/tabHeader.js'
  webcomponent: true,
  tabNav: { showAsTabHeader : true },
  children: [
  ...
```

> - **since**: 2.0.0

Reviewer: ```suggestion
```
